src/web/mod.rs

```diff
@@ -11,10 +11,11 @@ macro_rules! ctry {
         match $result {
             Ok(v) => v,
             Err(e) => {
-                return $crate::web::page::Page::new(format!("{:?}", e))
-                    .title("An error has occured")
+                log::error!("{}\n{:?}", e, backtrace::Backtrace::new());
+                return $crate::web::page::Page::new(format!("{}", e))
```

Comment:
Are you sure the user should get the raw error?

Reply:
I don't see why not, usually it's something like 'File not found' which seems pretty harmless.